addOns/pscanrules/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+Update Vulnerabilities' references to use https links for Loosely Scoped Cookie (Issue 8262).
```

Comment:
This should go under the existing Changed entry (e.g. in the same line as the other scan rules).